source/simulator/stokes_matrix_free.cc

```diff
@@ -1218,9 +1228,8 @@ namespace aspect
     sim.current_constraints.set_zero (linearized_stokes_initial_guess);
     linearized_stokes_initial_guess.block (block_p) /= sim.pressure_scaling;
 
-
     double solver_tolerance = 0;
-    if (sim.assemble_newton_stokes_system == false)
+    if (!sim.assemble_newton_stokes_system)
```

Comment:
nah, let's stick with the system we had -- that's what we do almost everywhere. it's just easier to read

Reply:
Ok, I will change the ones in `simulator/solver.cc`. This one here was changed to be consistent with the others.